build.gradle

```diff
@@ -0,0 +1,85 @@
+buildscript {
+    repositories {
+        mavenCentral()
+        maven {
+            name = "forge"
+            url = "http://files.minecraftforge.net/maven"
+        }
+        maven {
+            name = "sonatype"
+            url = "https://oss.sonatype.org/content/repositories/snapshots/"
+        }
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+    dependencies {
+        classpath 'net.minecraftforge.gradle:ForgeGradle:1.2-SNAPSHOT'
+        classpath "gradle.plugin.de.set.gradle:gradle-eclipse-compiler-plugin:1.4.1"
+    }
+}
+
+apply plugin: 'forge'
+apply plugin: "de.set.ecj"
+
+version = version
+group= "lembas"
+archivesBaseName = "lembas-core"
+
+srcCompat = JavaVersion.VERSION_1_8
+targetCompat = JavaVersion.VERSION_1_8
+
+minecraft {
+    version = "1.7.10-10.13.4.1614-1.7.10"
+    runDir = "run"
+
+    replaceIn "src/main/java/lembas/core/api/LembasCore.java"
+    replace "@version@", version
+}
+
+configurations {
```

Comment:
For the simplicity I used this one, the idea is to keep the file simple as long as we don't need the extended features of it, because it perturbates the whole build cycle

Reply:
I believe that using the shadow plugin would be simpler than this approach, but for now it is not that important.